rhcephcompose/main.py

```diff
@@ -25,6 +28,8 @@ def __init__(self):
             from requests.packages.urllib3.exceptions\
                 import InsecureRequestWarning
             requests.packages.urllib3.disable_warnings(InsecureRequestWarning)
+        if args.compose_type:
```

Comment:
We don't need to check this `if args.compose_type:` condition, right? It should always be set (particularly since we always write it in the format string later on.) I tested dropping the condition and it works ok.